myapp/tests/test_scheduler.py

```diff
@@ -0,0 +1,16 @@
+from django.test import TestCase
+from ..scheduler import scheduler
+import threading
+
+
+
+class test_scheduler(TestCase):
+
+    def test_scheduler_thread(self):
+        one = threading.active_count()
+        job = scheduler()
+        job.start()
+        self.assertNotEqual(one, threading.active_count())
+        job.stop()
```

Comment:
this is not Working! (doesn't stop the threads)
please make comprehensive testing on your code before creating the PR!

Reply:
i will check it again